Source/WebCore/dom/Text.cpp

```diff
@@ -104,13 +104,12 @@ static const Text* latestLogicallyAdjacentTextNode(const Text* text)
 
 String Text::wholeText() const
 {
-    const Text* startText = earliestLogicallyAdjacentTextNode(this);
-    const Text* endText = latestLogicallyAdjacentTextNode(this);
-    ASSERT(endText);
-    const Node* onePastEndText = TextNodeTraversal::nextSibling(*endText);
+    RefPtr startText = const_cast<Text*>(earliestLogicallyAdjacentTextNode(this));
```

Comment:
I believe we don't need the `const_cast<Text*>` here. You should be able to just do:

`RefPtr startText = earliestLogicallyAdjacentTextNode(this);`

Same goes for the other uses of `const_cast` here--could remove all of them.